.gitignore

```diff
@@ -11,3 +11,5 @@
 /composer.lock
 /phpunit.xml
 /google-service-account.json
+
+.idea
```

Comment:
I usually recommend having this in a global .gitignore file (together with OS- and other editor-specific files), so that the local file is really just project related 🙈

Reply:
You're right. I should do that :o